index.html

```diff
@@ -1,138 +1,215 @@
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="list">
+            <li class="item">css</li>
+            <li class="item">html</li>
+            <li class="item">bootstrap</li>
+            <li class="Ruby">Ruby</li>
+          </ul>
+          <button type="button" class="btn">See Project</button>
         </div>
-        <div class="background4 background">
-            <div class="stories1">
-                <img src="images/placeholder.png" alt="placeholder">
-            </div>
-            <div class="holder">
-                <h2>Multi-Post Stories</h2>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-                <ul class="list">
-                    <li class="item">css</li>
-                    <li class="item">html</li>
-                    <li class="item">bootstrap</li>
-                    <li class="Ruby">Ruby</li>
-                </ul>
-                <button type="button" class="btn">See Project</button>
-            </div>
+      </div>
+      <div class="background4 background">
+        <div class="stories1">
+          <img src="images/placeholder.png" alt="placeholder" />
         </div>
+        <div class="holder">
+          <h2>Multi-Post Stories</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="list">
+            <li class="item">css</li>
+            <li class="item">html</li>
+            <li class="item">bootstrap</li>
+            <li class="Ruby">Ruby</li>
+          </ul>
+          <button type="button" class="btn">See Project</button>
+        </div>
+      </div>
     </section>
+    <section>
+        <div class="contact">
 
-</body>
-</html>
\ No newline at end of file
+            <h1>CONTACT US</h1>
+            <p>
+              If you have an application you are interested in developing, a feature
+              that you need built or a project that needs coding. I'd love to help
+              with it
+            </p>
+            <form class="form" action="https://formspree.io/f/xlezygkr" method="POST">
+              <input id="name" name="Name" class="text" type="text" placeholder="Text Input" maxlength="30" required>
+                  <input id="email" name="email" class="text" type="email"  placeholder="example@gmail.com" required>
+                  <textarea name="Message" id="message" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+                  <button type="submit" class="btn" id="btn">See Project</button>
```

Comment:
- [ ] Your Button should have interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.